grails-app/controllers/org/pih/warehouse/inventory/InventoryController.groovy

```diff
@@ -253,9 +253,11 @@ class InventoryController {
     def search = { QuantityOnHandReportCommand command ->
         def quantityMapByDate = [:]
         def startTime = System.currentTimeMillis()
+        def startDate = new Date(command.startDate.getTime() + Calendar.getInstance().getTimeZone().getOffset(command.startDate.getTime()));
```

Comment:
I don't want to do this every time we have to take timezone into account. 
1) What exactly are we trying to solve here
2) Is there a more universal way of dealing with this